week-05/README.md

```diff
@@ -15,53 +15,56 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 5, Problem 1]
-
+- Question (lsun20-stat6250): Why we need to use quotation mark in this statement?
 
 
 [Course Textbook Chapter 5, Problem 2]
-
+- Question (lsun20-stat6250): Is there any other statement like FILENAME statement are global?
 
 
 [Course Textbook Chapter 5, Problem 6]
+- Question (lsun20-stat6250): What is the usage of the dollar sign($)?
 
-
+- Answer (lsun20-stat6250): The dollar sign($) is used to identify the character variables.
 
 [Course Textbook Chapter 5, Problem 7]
+- Question (lsun20-stat6250): Is there any order of input these column?
 
-
+- Answer (lsun20-stat6250): We can use column input to read fields in any order.
 
 [Course Textbook Chapter 5, Problem 8]
-
+- Question (lsun20-stat6250): Can we change the statement to "income*2=income" in this situation?
 
 
 [Course Textbook Chapter 6, Problem 1]
+- Question (lsun20-stat6250): What is the data set descriptor include?
 
-
+- Answer (lsun20-stat6250): The description includes the name of the data set, the number of observations and variables and the names and attributes of the variables.
 
 [Course Textbook Chapter 6, Problem 2]
-
+- Question (lsun20-stat6250): What are the things need to be considered a syntax error? list them.
```

Comment:
Please ask questions you actually have; you're not making a quiz for someone to take.